03_launch_mgmt_cluster.sh

```diff
@@ -185,49 +185,46 @@ EOF
     echo "IRONIC_KERNEL_PARAMS=console=ttyS0" | sudo tee -a "$IRONIC_DATA_DIR/ironic_bmo_configmap.env"
   fi
 
+  # Copy the generated configmap for ironic deployment
+  cp "$IRONIC_DATA_DIR/ironic_bmo_configmap.env"  "${BMOPATH}/ironic-deployment/keepalived/ironic_bmo_configmap.env"
+  # Backup Ironic manifests before changing the images in them.
+  # These are cleaned up in 05_test.sh
+  cp "${BMOPATH}/ironic-deployment/default/kustomization.yaml" "${BMOPATH}/ironic-deployment/default/kustomization.yaml.orig"
+  cp "${BMOPATH}/ironic-deployment/keepalived/kustomization.yaml" "${BMOPATH}/ironic-deployment/keepalived/kustomization.yaml.orig"
```

Comment:
Sorry, I did not make this clear. I'm restoring them in [05_test.sh](https://github.com/Nordix/metal3-dev-env/blob/lentzi90/fix-pivot-config/05_test.sh#L36-L47).

The logic is that we will need them for pivot, repivot and upgrade (possibly some other things as well), so to avoid a lot of confusion and extra steps I'm moving the clean up to the last part. It is quite hard to know where these changes are needed, for example they are also affecting e2e tests run from CAPM3 so I think it is better this way.

This means that if for some reason a developer does not run `05_test.sh`, then the repos will be dirty. It is not ideal but I don't see a better alternative.

Reply:
I see less point of keeping this file restoration now at this point, if we need the files to be as it is throughout the test, lets just completely remove this copy and restoration as you were doing before, sorry for pinging you back and forth but the current version doesnt seem logical to me.

Reply:
Ok, no problem! I'll remove it completely and also create an issue to investigate a better solution for this in the future. :slightly_smiling_face:

Reply:
@kashifest done, please take a look! Note: I put it as a separate commit for now so it is easy to change. If it looks good to you I'll squash

Reply:
@lentzi90 can you edit the description accordingly I think "For this reason I have moved the
cleanup to the 05_test.sh script." is not true anymore /lgtm